py/dml/codegen.py

```diff
@@ -2005,6 +2021,8 @@ def foreach_constant_list(site, itername, lst, statement, location, scope):
                     stmt)
             spec.append(mkCompound(site, decls + [stmt]))
 
+        if site.dml_version() != (1, 2):
+            spec.append(mkUnrolledLoopLabel(site, unroll.id))
```

Comment:
Selectively generating the label is actually easier than I first thought; I can have `stmt_break` mutate a `used` variable of the `UnrolledLoopContext` item in the `loop_stack`. So it's straight-forward to eliminate the label for `#foreach` loops that don't use `break` at all.
I don't want to drop the `UNUSED` annotation though, because I'm concerned some code elimination shenanigans might drop a break statement, even though `stmt_break` is called.

Reply:
Done.